File: app/src/main/java/com/wuruoye/know/util/sql/SqliteHelper.java
package com.wuruoye.know.util.sql;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import com.wuruoye.know.util.sql.table.RecordTable;
import com.wuruoye.know.util.sql.table.RecordTypeItemTable;
import com.wuruoye.know.util.sql.table.RecordTypeTable;
import com.wuruoye.know.util.sql.table.TextViewTable;

public class SqliteHelper extends SQLiteOpenHelper {
    public static final int VERSION = 1;
    public static final String DB_NAME = "con.wuruoye.know.db";

    public SqliteHelper(Context context) {
        super(context, DB_NAME, null, VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        TextViewTable.create(db);
        RecordTypeTable.create(db);
        RecordTable.create(db);
        RecordTypeItemTable.create(db);
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: app/src/main/java/com/wuruoye/know/util/sql/SqliteHelper.kt
package com.wuruoye.know.util.sql

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import com.wuruoye.know.util.sql.table.*

class SqliteHelper(context: Context) :
        SQLiteOpenHelper(context, DB_NAME, null, VERSION) {

    override fun onCreate(db: SQLiteDatabase) {
        TextViewTable.create(db)
        LayoutViewTable.create(db)
        RecordTypeTable.create(db)
        RecordTable.create(db)
        RecordItemTable.create(db)
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {

    }

    companion object {
        val VERSION = 1
        val DB_NAME = "con.wuruoye.know.db"
    }
}
